resources/views/processes/modeler/index.blade.php

```diff
@@ -39,30 +26,41 @@
   height: 100%;
   max-height: 100%;
 }
-
-[aria-label="breadcrumb"] {
-  position: relative;
-}
-
-.modeler-save-button {
-  right: 1rem;
-  transform: translateY(-50%);
-  top: 50%;
-}
 </style>
 @endsection
 
 @section('js')
   <script src="{{mix('js/leave-warning.js')}}"></script>
-  <script>
+  <script> 
+  const breadcrumbData = [
+    {
+      'text':'{{__('Designer')}}',
+      'url':'{{route('processes.index')}}'
+    },
+    {
+      'text':'{{__('Processes')}}',
+      'url':'{{route('processes.index')}}'
+    },
+    {
+      'text':'{{__('Modeler')}}',
+      'url':''
+    },
+    {
+      'text':'{{$process->name}}',
+      'url':''
+    },
+  ]
   window.ProcessMaker.modeler = {
     process: @json($process),
-    xml: @json($process->bpmn)
+    xml: @json($process->bpmn),
+    processName: @json($process->name),
   }
   const warnings = @json($process->warnings);
-  window.ProcessMaker.EventBus.$on('modeler-start', ({ loadXML, addWarnings }) => {
+ 
+  window.ProcessMaker.EventBus.$on('modeler-start', ({ loadXML, addWarnings, breadcrumbs }) => {
     loadXML(window.ProcessMaker.modeler.xml);
     addWarnings(warnings || []);
+    breadcrumbs(breadcrumbData || []);
```

Comment:
Do you think a name like `addBreadcrumbs` would be more readable? I think it will keep it consistent with the naming for the function above it (`addWarnigns`) and also better communicate what the function is actually doing.